doc/ref_model/chapters/chapter02.md

```diff
@@ -13,12 +12,12 @@ One of the main targets of the CNTT is to define an agnostic cloud infrastructur
 
 This means, operators will be able to host their Telco workloads (VNFs/CNFs) with different traffic types, behaviour and from any vendor on a unified consistent cloud infrastructure.
 
-Additionally, a well defined cloud infrastructure is also needed for other type of workloads such as IT, Machine learning, Artificial Intelligence, etc.
+Additionally, a well defined cloud infrastructure is also needed for other type of workloads such as IT, Machine Learning, Artificial Intelligence, etc.
```

Comment:
@walterkozlowski Since the sentence states, "such as", may I suggest the following:
```suggestion
Additionally, a well defined cloud infrastructure is also needed for other type of workloads such as IT, Machine Learning, and Artificial Intelligence.
```